experimental/await-ops.md

```diff
@@ -0,0 +1,22 @@
+# [Await.ops](https://github.com/tc39/proposal-await.ops)
+
+## AwaitExpression
+
+```js
+interface AwaitExpression <: Expression {
+    type: "AwaitExpression";
+    argument: Expression;
+    operation: "all" | "any" | "race" | "allSettled" | undefined
```

Comment:
To allow easy extensibility, should this be an `Identifier | null` so future additions won’t have to change the AST or does it make sense to strictly limit the allowed operations?

Reply:
In the typescript implementation of this [(repl)](https://www.staging-typescript.org/play?ts=4.0.0-pr-39224-4#code/IYZwngdgxgBAZgV2gFwJYHsIwB4AoCUMA3gFAzkzADuwqyAdMADZMwAMZF1tDATsFACmMANoAFXugC2qEIPq9BIdEwBug3AEZ8AXU7ludRiwDKg5MiaCAJqL0BfIA), it does throw an error so I think we need it to be strict. 

Though I don't have strong opinion on either

Reply:
It seems to me that current spec disallows whitespace in `await.all`, which TypeScript implementation currently allow
```js
await
// some comment
   .all [Promise.resolve(1)]
```
then an enum looks good to me. But I believe the spec should allow whitespaces. I have submitted https://github.com/tc39/proposal-await.ops/issues/12.

If whitespaces are allowed, an `Identifier | null` will offer position informations for tools. We had similar discussion about that on `MetaProperty` ([thread](https://github.com/estree/estree/pull/32#issuecomment-77819820)).

Reply:
Also agree that an enum is most appropriate here as anything other than these values would be a syntax error.

Reply:
Sorry for the wordings before. I lean to a dedicated Identifier for tracking positional information of `all` _if_ whitespaces are allowed: https://github.com/tc39/proposal-await.ops/pull/13

Reply:
+1 for enum. But, either way, let's remember this is an experimental PR ;)

Reply:
Updated the PR with `Indentifier`